e2e/tests/plugin-form-Nested.spec.ts

```diff
@@ -93,28 +93,47 @@ test('Adding a trainee', async () => {
   ).toHaveValue('123')
 })
 
+test('Locations', async () => {
+  const locations = page.getByTestId('locations')
+  await expect(locations.getByRole('textbox')).toHaveCount(1)
+  await expect(locations.getByRole('textbox')).toHaveValue('Trondheim')
+  await locations.getByRole('button', { name: 'Add' }).click()
+  await expect(locations.getByRole('textbox')).toHaveCount(2)
+  await locations.getByRole('textbox').last().fill('Oslo')
+  await page.getByRole('button', { name: 'Submit' }).click()
+  await page.reload()
+  await navigate()
+  await expect(locations.getByRole('textbox')).toHaveCount(2)
+  await expect(locations.getByRole('textbox').first()).toHaveValue('Trondheim')
+  await expect(locations.getByRole('textbox').last()).toHaveValue('Oslo')
+  await locations.getByRole('button', { name: 'Remove' }).last().click()
+  await expect(locations.getByRole('textbox')).toHaveCount(1)
+  await page.getByRole('button', { name: 'Submit' }).click()
+  await page.reload()
+  await navigate()
+  await expect(locations.getByRole('textbox')).toHaveCount(1)
+})
+
 test('New car', async () => {
-  await page.getByText('CarsOpen').getByRole('button', { name: 'Open' }).click()
-  await expect.soft(page.getByText('1 - 2 of 2')).toBeVisible()
-  await page.getByRole('button', { name: 'Append Add Item' }).click()
-  await expect.soft(page.getByText('1 - 3 of 3')).toBeVisible()
-  await page.getByRole('button', { name: 'Save' }).click()
-  await page.getByRole('button', { name: 'Open item' }).last().click()
+  const cars = page.getByTestId('cars')
```

Comment:
```suggestion
  const carsPageElement = page.getByTestId('cars')
```

maybe rename the variable to something a bit more descriptive. Just an example name provided.

Reply:
I went for carsDiv - short and descriptive. I also renamed the locations variable in the same manner.